homeassistant/components/plugwise/sensor.py

```diff
@@ -327,6 +334,9 @@ def _async_process_data(self):
         if data.get("cooling_state") is not None:
             self._cooling_state = data["cooling_state"]
 
+        if CUSTOM_ICONS.get(self._sensor) is not None:
+            self._icon = CUSTOM_ICONS.get(self._sensor)
+
         self._state = "idle"
         self._icon = IDLE_ICON
```

Comment:
Here we overwrite the icon we just set above. How do we want this to work?

Reply:
Should first take cafeine, than apply code :( sorry about that.